### PR TITLE
config modification

### PR DESCRIPTION
**Pull request by donghwipark**
1. Plugins : isProd seems occuring error
2. Color mode change
3. navbar parameter naming
4. Footer title miss

### CODE REVIEW

**Review by phated — CHANGES_REQUESTED** [review]
@donghwipark when you change all the `'` to `"`, I can't tell which changes actually matter. Can you please switch those back to `'`?

**Comment by donghwipark** [comment]
Oh it seems I accidently changed through prettier. Will change to " to ' and mention here which code I changed manually.

**Comment by donghwipark** [comment]
I changed to single quote and seem you can check in fils changed exactly which has modified please confirm. 👍 
https://github.com/gulpjs/gulpjs.github.io/pull/131/files

**Comment by phated** [comment]
Thank you for fixing that! I notice that it added commas also, but I'm those should be there anyway. I will review this and make suggestions this weekend.

**Comment by phated** [comment]
@donghwipark I just noticed that you have not committed any changes to the `package.json` and `package-lock.json` files, which need to be updated for the new version of docusaurus. Did you mistakenly leave out those updates?

**Comment by donghwipark** [comment]
It was a mistake, will pull request with package json

File: docusaurus.config.js [hunk 1]
```diff
@@ -177,29 +184,35 @@ module.exports = {
...
+    ],
+    'docusaurus-plugin-sass',
     ['@docusaurus/plugin-content-pages', {}],
-    isProd && ['@docusaurus/plugin-google-analytics', {}],
-    isProd && ['@docusaurus/plugin-google-gtag', {}],
-    isProd && ['@docusaurus/plugin-sitemap', {}],
-  ]
+    ['@docusaurus/plugin-google-analytics', {}],
+    ['@docusaurus/plugin-google-gtag', {}],
+    ['@docusaurus/plugin-sitemap', {}],
```
Comment (phated) [inline docusaurus.config.js:216]: Version `alpha.68` and higher fixed this issue (with https://github.com/facebook/docusaurus/pull/3704), so we can add these back.
```suggestion
    isProd && ['@docusaurus/plugin-google-analytics', {}],
    isProd && ['@docusaurus/plugin-google-gtag', {}],
    isProd && ['@docusaurus/plugin-sitemap', {}],
```

**Review by phated — CHANGES_REQUESTED** [review]
@donghwipark Sorry it has taken so long to review this. It looks like version `alpha.70` has been released since you submitted this PR. Can you please update and apply any changes needed for that new version?

**Comment by phated** [comment]
@donghwipark it looks like you pulled/merged master instead of the `source` branch 😬

**Comment by donghwipark** [comment]
Yes I accidently pulled the master.. I pull request with another commit please confirm.

**Pull request closed without merging**